### PR TITLE
readme: Guide for installing on fresh Windows Subsystem for Linux

### PR DESCRIPTION
**Pull request by denis-bogdanas**

### CODE REVIEW

**Comment by ehildenb** [comment]
@DustinWehr ping with regards to #389

**Review by ehildenb — APPROVED** [review]
I'm approving, would like Dustin's feedback to make sure it works for him too, but I don't think we should block merging on it.

**Comment by denis-bogdanas** [comment]
I modified his instructions for what worked for me. There are some differences. I didn't add instructions creating a new user. Just for running with root.

**Comment by DustinWehr** [comment]
Looks good! An issue for me with running as root is getting 
> [WARNING] Running as root is not recommended

with every use of kompile, which is worrisome. It led me to googling about WSL security, which took a lot longer than simply changing the permissions. Whether it actually helps or not, it was the shortest path to obtaining the minimal comfort I require.

**Comment by denis-bogdanas** [comment]
@ehildenb Are you ok merging like this? I guess there are cleaner ways to install K on non-root user than what's in Dustin's guide. But this is a good enough starter.

I'd rather not spend time on verifying how to install on a different user.

**Comment by ehildenb** [comment]
Let's just add a notice at the beginning which says "These instructions are for root user, make sure to create a non-root user to run as if you care not to do that" or something.

**Comment by denis-bogdanas** [comment]
@ehildenb I added a new point with links to guides how to create a new user. Adding that sentence at the beginning might get confusing.

File: README.md
Comment (dwightguth) [inline README.md]: the first sentence here is not true.

File: README.md
Comment (dwightguth) [inline README.md]: this will not work as this directory will be owned by root. You need to tell people to `cp -R /usr/lib/kframework/tutorial ~` and then tell them to build the version of the tutorial they copied into their home directory.

**Review by dwightguth — CHANGES_REQUESTED** [review]

**Comment by denis-bogdanas** [comment]
@dwightguth fixed. Added the command to change owner, like Dustin din.

**Comment by dwightguth** [comment]
As I said, the default WSL user is /not/ root and we should not be telling people that because it will just confuse them. Nor should we be telling them to chown the installation as that is not necessary.

**Comment by denis-bogdanas** [comment]
@dwightguth The way I installed it today, from `WSLTools` utility, it is root. But if one uses the install wizard from windows store, it probably prompts to create a new user. So you suggest deleting the whole part about creating/changing user?

I'll change the other part.

**Comment by dwightguth** [comment]
We should definitely give people the official Microsoft-supported instructions for installing WSL.

**Comment by denis-bogdanas** [comment]
done

File: README.md
Comment (dwightguth) [inline README.md]: I think you meant for there to be a cd here.

**Review by dwightguth — APPROVED** [review]
looks good but one minor thing to fix before you merge.

File: README.md
Comment (denis-bogdanas) [reply README.md]: done

File: README.md
Comment (DustinWehr) [inline README.md:119]: Heh, ah, yeah that's better than chown.

**Pull request merged**
